FirebaseVertexAI/Sources/GenerativeModel.swift

```diff
@@ -50,8 +50,7 @@ public final class GenerativeModel {
   /// Initializes a new remote model with the given parameters.
   ///
   /// - Parameters:
-  ///   - name: The name of the model to use, for example `"gemini-1.0-pro"`; see
-  ///     [Gemini models](https://ai.google.dev/models/gemini) for a list of supported model names.
+  ///   - name: The name of the model to use, for example `"gemini-1.0-pro"`.
```

Comment:
Should this line be updated to something newer?

Reply:
I debated whether to change this one but thought it might be best to leave it as a stable model for now since it's more likely to be forgotten in docs updates since this constructor is internal.